express/blocks/shared/video.js

```diff
@@ -17,6 +17,22 @@ import {
 
 const docTitle = document.title;
 
+async function fetchVideoAnalytics() {
+  if (!window.videoAnalytics) {
+    window.videoAnalytics = [];
+    try {
+      const resp = await fetch('/express/video-analytics.json');
```

Comment:
I think this should be made localizable (see how we do it for promotions on L40-L42 below).